mulesoft/docs-site-ui
# [W-10592504] add composer login

ref: W-10592504

### Enhancements

- add composer login to the header (cmd + f `<a href="https://composer.mulesoft.com/login/sign-in">Composer</a>` to find the change. Nothing else has changed)

### How to validate

- run `gulp preview` on this branch
- open http://localhost:5252 in a web browser
- hover on **Login** on the upper right corner of the page, and click **Composer**
- it should navigate you to https://composer.mulesoft.com/login/sign-in on the same page (same behavior as mulesoft.com)
